neps/search_spaces/search_space.py

```diff
@@ -177,7 +183,7 @@ def pipeline_space_from_yaml(
                     "For categorical parameter: cat, categorical\n"
                     "For constant parameter: const, constant\n"
                 )
-    except (KeyError, TypeError, ValueError) as e:
+    except (KeyError, TypeError, ValueError, FileNotFoundError) as e:
         raise SearchSpaceFromYamlFileError(e) from e
     return pipeline_space
 
```

Comment:
This whole block can really just be a series of `try: excepts` rather than a nest `try: except`. No need to change now, just for your info